### PR TITLE
Temporary: try continuing even if crawl fails

### PR DESCRIPTION
**Pull request by chosak**
Not sure why this step [is returning exit code 3](https://github.com/cfpb/crawl-cfgov/runs/1229851189?check_suite_focus=true) even though the crawl seems to succeed:

```
FINISHED --2020-10-09 10:36:11--
Total wall clock time: 4h 15m 50s
Downloaded: 12368 files, 1.8G in 2h 22m 0s (225 KB/s)

real	255m50.444s
user	0m29.428s
sys	0m16.115s
Error: Process completed with exit code 3.
```

Let's experiment with [`continue-on-error`](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error).

### CODE REVIEW

File: .github/workflows/crawl.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ jobs:
 
       - name: Run the crawl script
         run: ./crawl.sh
+        continue-on-error: true
```
Comment (schbetsy) [inline .github/workflows/crawl.yml:20]: I think the current crawl code accomplishes the same thing with [this line](https://github.com/cfpb/crawl-cfgov/blob/main/.github/workflows/crawl.yml#L24). It now continues the job even if the crawl task exits with an error code.

File: crawl.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ time wget \
     --reject '*.css,*.doc,*.docx,*.gif,*.ico,*.jpg,*.js,*.mp3,*.pdf,*.png,*.txt,*.wav,*.woff,*.woff2,*.xls,*xlsx,*.zip' \
     --reject-regex "topics=|authors=|categories=|filter_blog_category=|ext_url=|search_field=|issuer_name=" \
     --recursive \
-    --level=inf \
+    --level=140 \
```
Comment (schbetsy) [inline crawl.sh:11]: I'm hoping to slowly step up the crawl level in a separate PR. We've never had success with anything other than level 3, so I think I'll try level 4 or 5 today.

**Review by schbetsy — COMMENTED** [review]
Sorry I missed this PR when you opened it. Okay to close?

**Comment by chosak** [comment]
Thanks @schbetsy, closing as suggested.

**Pull request closed without merging**